File: MiniDOMWebCtrls/DMD/CQSPD/CAmministrazioniModuleHandler.cs

namespace minidom.Forms
{
    public class CAmministrazioniModuleHandler : CPersoneGiuridicheModuleHandler
    {
        public override Databases.DBObjectCursorBase CreateCursor()
        {
            var ret = new Anagrafica.CPersonaCursor();
            ret.TipoPersona.Value = Anagrafica.TipoPersona.PERSONA_GIURIDICA;
            return ret;
        }
    }
}

File: MiniDOMWebCtrls/DMD/CQSPD/CAmministrazioniModuleHandler.vb
Imports minidom
Imports minidom.Forms
Imports minidom.Databases
Imports minidom.WebSite

Imports minidom.Finanziaria
Imports minidom.Sistema
Imports minidom.Anagrafica

Namespace Forms
  

    Public Class CAmministrazioniModuleHandler
        Inherits CPersoneGiuridicheModuleHandler

        Public Overrides Function CreateCursor() As DBObjectCursorBase
            Dim ret As New CPersonaCursor
            ret.TipoPersona.Value = TipoPersona.PERSONA_GIURIDICA
            Return ret
        End Function




    End Class


End Namespace